.gitignore

```diff
@@ -21,4 +21,3 @@ env/
 staticfiles/
 .DS_Store
 static/
-api/migrations
```

Comment:
When we make our first major deployment, this would be removed and we can then be editing migrations script instead.

Reply:
The question is... since I made some migrations to complete my task, how would other collaborators get those migrations if the migrations folder is ignored?